packages/lab/src/date-input/DateInput.css

```diff
@@ -90,7 +90,7 @@
 }
 
 .saltDateInput-bordered.saltDateInput-focused .saltDateInput-activationIndicator {
-  border-bottom-width: calc(var(--salt-size-border-strong) - var(--salt-size-border));
+  border-bottom-width: calc(var(--salt-size-fixed-200) - var(--salt-size-fixed-100));
```

Comment:
Doesn't this calc cancel each other out? Shouldn't it just be `--salt-size-fixed-100`?

Reply:
They do cancel each other out, this was mainly showing that it's the activation indicator width - the border width. Happy to simplify it if neccessary